test/integration/admin_login_test.rb

```diff
@@ -211,6 +211,8 @@ def setup
     click_link_or_button('Retire')
 
     visit items_path
-    refute page.has_content?('black nectar')    
+    refute page.has_content?('black nectar')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.